apps/mobile/src/features/Notifications/Notifications.container.tsx

```diff
@@ -1,10 +1,47 @@
+import React, { useCallback } from 'react'
+import { Switch } from 'react-native'
 import { View, Text } from 'tamagui'
-import React from 'react'
+
+import { useAppSelector, useAppDispatch } from '@/src/store/hooks'
+import { SafeListItem } from '@/src/components/SafeListItem'
+import {
+  selectAppNotificationStatus,
+  selectDeviceNotificationStatus,
+  toggleAppNotifications,
+} from '@/src/store/notificationsSlice'
 
 export const NotificationsContainer = () => {
+  const dispatch = useAppDispatch()
+  const isAppNotificationEnabled = useAppSelector(selectAppNotificationStatus)
+  const isDeviceNotificationStatus = useAppSelector(selectDeviceNotificationStatus)
```

Comment:
A nit, but what is this variable supposed to convey? `is` is generally used for boolean variables. Are we dealing with a boolean here?